src/pages/hegel/reference/The Mechanical Object: Section 1

```diff
@@ -0,0 +1,18 @@
+# The Mechanical Object – Section 1
```

Comment:
It looks you've uploaded two nearly identical files. This one should be deleted
![image](https://github.com/systemphil/sphil/assets/118931755/159f1c32-44ef-4327-81db-9fb080306583)

Reply:
I chose to do a break-down of my articles in accordance with the way that Hegel breaks up the chapter. So, in Mechanism, he begins with that five line preliminary paragraph where he introduces the mechanical object. That is the focus of the published article "The Mechanical Object First Paragraph". I chose to do this because (a) this is how I generally approach breaking-down Hegel's text and (b) I think it's a helpful way to follow the text because Hegel's break-down has a rationale behind it that makes the logical development easier to follow.